src/libs/actions/IOU.js

```diff
@@ -143,6 +143,7 @@ function createIOUSplit(params) {
             }
 
             chatReportID = response.reportID;
+            // eslint-disable-next-line rulesdir/no-multiple-api-calls
```

Comment:
@mountiny is actually removing the violation for `DeprecatedAPI` here: https://github.com/Expensify/eslint-config-expensify/pull/53/files#diff-936b4e8282631c0051d8abc7077a077a6648d17e47d84242963fd1d4c5170a4aR11

So perhaps we can wait for that change to be merged (soon) and we won't need these suppressions?